.circleci/config.yml

```diff
@@ -180,6 +180,15 @@ jobs:
     steps:
       - checkout
       - run: deployment/molecule/molecule.sh monitor
+  deployment-repo:
+    executor: tokenbridge-orb/machine-with-docker-caching
+    steps:
+      - checkout
+      - run: git submodule update --init
+      - tokenbridge-orb/install-node
```

Comment:
Seems I need an extended description what exactly we are solving by these changes. Could you provide it?

Reply:
1. `install-node` and `install-yarn` is needed to perform `yarn-install-cached-on-machine`
2. `yarn-install-cached-on-machine` is needed so that there exists `node_modules`
3. Existing `node_modules` is needed so that we can test, that the new way of deployment will not copy local `node_modules` to remote server